utils/constants/objaverse_data_dirs.py

```diff
@@ -28,9 +28,9 @@
 else:
     OBJAVERSE_HOUSES_DIR = os.path.abspath(OBJAVERSE_HOUSES_DIR)
 
-print(
-    f"Using"
-    f" '{OBJAVERSE_ASSETS_DIR}' for objaverse assets,"
-    f" '{OBJAVERSE_ANNOTATIONS_PATH}' for objaverse annotations,"
-    f" '{OBJAVERSE_HOUSES_DIR}' for procthor-objaverse houses."
-)
+# print(
+#     f"Using"
+#     f" '{OBJAVERSE_ASSETS_DIR}' for objaverse assets,"
+#     f" '{OBJAVERSE_ANNOTATIONS_PATH}' for objaverse annotations,"
+#     f" '{OBJAVERSE_HOUSES_DIR}' for procthor-objaverse houses."
+# )
```

Comment:
should we have this printing back to indicate which assets people are using?
or should we just remove this entirely?

Reply:
It prints it too many times! :D That's why I commented it out